modules/ROOT/pages/installation-prerequisites.adoc

```diff
@@ -6,34 +6,71 @@ endif::[]
 
 To install Anypoint API Community Manager you will need:
 
-. An Anypoint admin user account.
-. A Salesforce API Community Manager organization admin user with a role.
-+
+== A Salesforce Admin User
+
+Installation requires a Salesforce API Community Manager organization admin user with a role.
+
 To update your user role and permissions:
 
-.. Log in to your Salesforce organization.
-.. In *Setup*, search for *Users* using the *Quick Find* box and click *Users*.
-.. Locate your username and click *Edit*.
-.. Set the *Role* for your account.
+. Log in to your Salesforce organization.
+. In *Setup*, search for *Users* using the *Quick Find* box and click *Users*.
+. Locate your username and click *Edit*.
+. Set the *Role* for your account.
 +
 This can be any role from the list, but cannot be none.
 +
 If you don't see any roles in the list, refer to the Salesforce knowledge article https://help.salesforce.com/articleView?language=en_US&type=1&mode=1&id=000322715[Add roles to the role hierarchy, role=external, window=_blank] for instructions to create a role in your organization.
+. Select *Salesforce CRM Content User*.
+. *Save*.
+
+== API Community Manager Entitlement
+
+Installation requires Anypoint API Community Manager entitlement for your Anypoint organization.
 
-.. Select *Salesforce CRM Content User*.
-.. Click *Save*.
-. The Anypoint API Community Manager entitlement for your Anypoint organization.
-+
 To check the entitlement:
 
-.. Log in to your Anypoint Platform organization.
-.. If you are using the US cloud, navigate to `+https://anypoint.mulesoft.com/accounts/api/profile+`.
+. Log in to your Anypoint Platform organization.
+. If you are using the US cloud, navigate to `+https://anypoint.mulesoft.com/accounts/api/profile+`.
 +
 If you are using the EU cloud, navigate to `+https://eu1.anypoint.mulesoft.com/accounts/api/profile+`.
 +
 If you are using MuleSoft Government Cloud, navigate to `+https://gov.anypoint.mulesoft.com/accounts/api/profile+`.
```

Comment:
This is not the proper format for a step with options.
Create a general step, ending with a colon, and present the options as bullets.